collections/_standard/masthead.html

```diff
@@ -1,17 +1,16 @@
 ---
 title: Masthead
-subtitle: Subtitle
 layout: layout-docs
 category: Standard
 icon: masthead
-breadcrumb: Standard
 order: 1
-js: true
 ---
 
 <h4 class="has-text-primary">
-    The Masthead is a standard component that has to be present in all pages of
-    '.gov.sg' websites.
+    All .gov.sg digital services shall implement
+    the Official Government Banner
+    (containing the Singapore Government
```

Comment:
Maybe reword to (Which must include the Singapore Government Online Logo)?

Reply:
Think we should keep it since its taken from the DSS Standard 3.9